test/compilable/previewhelp.d

```diff
@@ -11,7 +11,6 @@ Upcoming language changes listed by -preview=name:
   =dip1021          implement https://github.com/dlang/DIPs/blob/master/DIPs/accepted/DIP1021.md (Mutable function arguments)
   =fieldwise        use fieldwise comparisons for struct equality
   =fixAliasThis     when a symbol is resolved, check alias this scope before going to upper scopes
-  =intpromote       fix integral promotions for unary + - ~ operators
```

Comment:
Instead of removing it why not mark it as deprecated?

Reply:
It is marked as deprecated in cli.d, but also as undocumented:
```D
Feature("intpromote", "fix16997",
            "fix integral promotions for unary + - ~ operators"),
            "fix integral promotions for unary + - ~ operators", false, true),
```
Which is what Iain suggested:

> You're not removing it, just explicitly undocumenting it, and optionally warning about its use.

No use in listing a useless switch.

Reply:
Right, there are two use cases:
1. `false, false` - this switch is redundant, so no need to document it anymore, however we accept that there may be many uses of in the wild and don't want to immediately break downstream builds just yet.
2. `false, true` - same as above, but start warning downstream projects that it is time to move off this switch.